.gitignore

```diff
@@ -4,6 +4,7 @@ node_modules
 *.log
 /coverage
 /build
+/dist
```

Comment:
Since we have moved from build to dist, can we remove `/build` from here?

Reply:
Yes, but someone remains build directory in their local machines